src/sources/dataModel/taxonomic-service.json

```diff
@@ -268,14 +539,37 @@
             "type": "object",
             "description": "A FundingScheme combines organizational, project and policy aspects of grant-based funding that sets guidelines, principles and mechanisms to support other kinds of projects and activities.",
             "properties": {
-                "schema:url": {
+                "schema:award": {
                     "type": "string",
-                    "description": "URL to webpage with the supported payment models and restrictions that apply to the Resource."
+                    "description": "An award won by or for this service",
+                    "examples": [
+                        "National Award 2024"
+                    ]
                 },
-                "schema:Funder": {
+                "schema:Funding": {
```

Comment:
Feels like this also needs a schema:identifier field

Reply:
Added it